internal/api/server/unix_socket.go

```diff
@@ -114,6 +114,8 @@ func handleUnixSocketConnection(connection net.Conn) error {
 		return fmt.Errorf("Failed to find the 'endpoint' key in the request.")
 	}
 
+	endpoint = core.MakeFullAPIPath(endpoint)
```

Comment:
The variable name (the input here) needs to change with the semantics (the output can stay the same).
You also need to add a comment so people known that they don't need to pass a full API path.
I'm thinking you can make the comment where `ENDPOINT_KEY` is defined.